ENDPOINT_STATUS.md

```diff
@@ -1,32 +1,35 @@
 # Status
 
-Check here to see if the library is known to be working or not against each DiGA endpoint, 
+Check here to see if the library is known to be working or not against each DiGA endpoint,
```

Comment:
Haha, yeah I guess it reads nicer as table and probably also easier to update. Wasnt sure to include it in this PR and might open a separate one. But given that the PR is focussed on documentation, I think it might be alright. When opening the PR, I can drop a comment to your format